packages/plugin-ext/src/hosted/node/scanners/plugin-uri-factory.ts

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
Extra whitespace to remove, surprised eslint didn't complain.

Reply:
Is the code more readable/safer/better without that line? If not, why enforce? But I digress....

Reply:
I see what you mean, but I still prefer well formatted files, again this is most likely an issue with eslint.